apps/namadillo/src/slices/fees.ts

```diff
@@ -12,11 +12,16 @@ export const minimumGasPriceAtom = atomWithQuery<BigNumber>((get) => {
   return {
     queryKey: ["minimum-gas-price", nativeToken],
     queryFn: async () => {
-      const { rpc } = await getSdkInstance();
-      const result = await rpc.queryGasCosts();
-      const nativeTokenCost = result.find(([token]) => token === nativeToken);
+      const api = new DefaultApi();
+      const gasTableResponse = await api.apiV1GasTableGet();
+
+      // TODO: Can nativeToken ever be undefined?
```

Comment:
Was just thinking about this, and I think we should fall back to a default address: `tnam1qxgfw7myv4dh0qna4hq0xdg6lx77fzl7dcem8h7e` from the point of view of the interface so it should never be undefined. So if it is not set in `.env` or the `queryNativeToken` call fails, I feel like `nativeToken` should still be defined in the app. Just a thought!

Reply:
Just to add, the interface currently won't load at all if the native token is not set in `.env`.

Reply:
👍

Reply:
yeah, I discussed with @pedrorezende a bit about this issue